src/azure-cli/azure/cli/command_modules/acr/task.py

```diff
@@ -982,3 +986,13 @@ def _get_trigger_event_list_patch(cmd,
             if SourceTriggerEvent.pullrequest.value in source_trigger_events:
                 source_trigger_events.remove(SourceTriggerEvent.pullrequest.value)
     return source_trigger_events
+
+
+def _get_branch_name(context_path):
```

Comment:
This might be overkill, but do we want to add a unit test or tests for this method? Especially, given it relies on regexes? It might work fine now, but what if someone modifies its behavior and breaks it in the future?

Reply:
we're going to update the backend side to handle this case and remove this code completely. So I don't think is worth the trouble for now.